pkg/backup/backup.go

```diff
@@ -279,10 +279,12 @@ func (kb *kubernetesBackupper) BackupWithResolvers(log logrus.FieldLogger,
 	items := collector.getAllItems()
 	log.WithField("progress", "").Infof("Collected %d items matching the backup spec from the Kubernetes API (actual number of items backed up may be more or less depending on velero.io/exclude-from-backup annotation, plugins returning additional related items to back up, etc.)", len(items))
 
-	backupRequest.Status.Progress = &velerov1api.BackupProgress{TotalItems: len(items)}
-	original := backupRequest.Backup.DeepCopy()
-	backupRequest.Backup.Status.Progress.TotalItems = len(items)
-	if err := kube.PatchResource(original, backupRequest.Backup, kb.kbClient); err != nil {
+	updated := backupRequest.Backup.DeepCopy()
+	if updated.Status.Progress == nil {
+		updated.Status.Progress = &velerov1api.BackupProgress{}
+	}
+	updated.Status.Progress.TotalItems = len(items)
+	if err := kube.PatchResource(backupRequest.Backup, updated, kb.kbClient); err != nil {
 		log.WithError(errors.WithStack((err))).Warn("Got error trying to update backup's status.progress.totalItems")
 	}
```

Comment:
This line should also update the `backupRequest.Status.Progress` value.

Reply:
Done